lib/js-preprocessor/index.js

```diff
@@ -8,7 +8,7 @@ var funnel = require('broccoli-funnel');
 module.exports = function(tree, options) {
   var generated = new GenerateComponents(tree, options);
 
-  var merged = mergeTrees([tree, generated]);
+  var merged = mergeTrees([tree, generated], { overwrite: true });
```

Comment:
I was running into issues when this wasn't set. What are the repercussions of this?